browser/main/Detail/TagSelect.styl

```diff
@@ -3,15 +3,12 @@
   align-items center
   user-select none
   vertical-align middle
-  width 100%
+  width 96%
   overflow-x scroll
```

Comment:
Can you change `overflow-x scroll` to `overflow-x auto` so the scroll bar is shown only when needed 😉

Reply:
![image](https://user-images.githubusercontent.com/43637878/50148421-6cf7d300-02f3-11e9-8c36-b172ef6610d5.png)
Even without `overflow-x auto`, the scroll bar will not show by default unless the tag list is overflow, not sure why the behavior is like this....

Reply:
Thank for the update.

With `overflow-x scroll`, I have:
![screenshot](https://user-images.githubusercontent.com/587742/50150205-8e3dcc80-02bd-11e9-8e78-2d4ba94eee79.png)
The space bellow is for the scrollbar but the thumb isn't yet shown

With `overflow-x auto`, I have:
![screenshot](https://user-images.githubusercontent.com/587742/50150206-8e3dcc80-02bd-11e9-945b-a60dde98d27a.png)
Same design as on the `master`.